src/containerapp/azext_containerapp/containerapp_decorator.py

```diff
@@ -1333,9 +1333,51 @@ def post_process(self, r):
                 while r is not None and r["properties"]["provisioningState"].lower() == "inprogress":
                     r = self.client.show(self.cmd, self.get_argument_resource_group_name(), self.get_argument_name())
                     time.sleep(1)
-                linker_client.linker.begin_create_or_update(resource_uri=r["id"],
-                                                            parameters=item["parameters"],
-                                                            linker_name=item["linker_name"]).result()
+
+                service_connectors_def_list, service_bindings_def_list = parse_service_bindings(
+                    self.cmd, self.get_argument_service_bindings(), self.get_argument_resource_group_name(), self.get_argument_name())
+                self.set_argument_service_connectors_def_list(
+                    service_connectors_def_list)
+
+                # Case: Kafka on confluent cloud with two linker operations.
+                # Validate uniqueness across all bindings
+                if any(len(item["parameters"]) ==
```

Comment:
is there any other index to distinguish the cases besides parameters length?

Reply:
i don't think so for this case...reason being that for Managed Kafka, the only difference from other managed services is that we are working with two linkers/links at the same time. so if it weren't len(parameters) it'd be len(linker_name) had we made the linker_name object a list.